docs/specification/objects.md

```diff
@@ -506,6 +508,7 @@ That is, the spec's process and the process's spec must point to the same Proces
 Field name | Relationship | Field Name
 -----------|:------------:|------------
 process.spec | = | spec.process
+process | must be unique | globally
```

Comment:
Since line 503 says that the constraints are the same as a Material Spec, this line is redundant.

That said, I'm not entirely opposed to redundancy.

Reply:
Nice catch.  I avoided explicitly including that in Process Run context, so consistency suggests I skip it here.